neo4j/_async/driver.py

```diff
@@ -392,6 +392,108 @@ async def supports_multi_db(self):
             await session._connect(READ_ACCESS)
             return session._connection.supports_multiple_databases
 
+    async def query(self, query, parameters=None, **kwargs):
+        """
+        Run a Cypher query within an managed transaction and
+        all the retries policy will be applied.
+
+        The query is sent and the result header received
+        immediately and the :class:`neo4j.QueryResult`is
+        fetched.
+
+        For more usage details, see :meth:`.AsyncTransaction.query`.
+
+        For auto-commit queries, use :meth:`AsyncSession.run`.
+
+        For access to the :class:`neo4j.AsyncResult` object,
+        use :meth:`AsyncDriver.execute` and :meth:`.AsyncTransaction.run`
+
+        :param query: cypher query
+        :type query: str, neo4j.Query
+        :param parameters: dictionary of parameters
+        :type parameters: dict
+        :param kwargs: additional keyword parameters
+
+        :returns: a new :class:`neo4j.QueryResult` object
+        :rtype: QueryResult
+        """
+        session_kwargs = {}
+        if "database" in kwargs:
+            session_kwargs["database"] = kwargs.pop("database")
+
+        async with self.session(**session_kwargs) as session:
+            return await session.query(query, parameters, **kwargs)
+
+    async def execute(self, transaction_function, *args, **kwargs):
+        """Execute a unit of work in a managed transaction.
+
+        .. note::
+            This does not necessarily imply access control, see the session
+            configuration option :ref:`default-access-mode-ref`.
+
+        This transaction will automatically be committed unless an exception
+        is thrown during query execution or by the user code.
+        Note, that this function perform retries and that the supplied `
+        transaction_function` might get invoked more than once.
+
+        Managed transactions should not generally be explicitly committed
+        (via ``tx.commit()``).
```

Comment:
```suggestion
```
It's funny that this is still in the docs :hand_over_mouth: in fact, they can't anymore. This method does no longer exist for managed transactions.